src/Angor/Avalonia/Angor.Wallet/Domain/BroadcastedTransaction.cs

```diff
@@ -0,0 +1,15 @@
+namespace Angor.Wallet.Domain;
+
+public record BroadcastedTransaction(
+    Balance Balance,
+    string Id,
+    IEnumerable<TransactionInputInfo> WalletInputs, 
+    IEnumerable<TransactionOutputInfo> WalletOutputs, 
+    IEnumerable<TransactionAddressInfo> AllInputs, 
+    IEnumerable<TransactionAddressInfo> AllOutputs, 
+    ulong Fee,
```

Comment:
We tend to represent sats as longs (not ulong) just to be consistent

Reply:
Will change that